### PR TITLE
[bitnami/charts] Automate license checks

### PR DESCRIPTION
**Pull request by rloporp**
### Description of the change

Licenses and copyright notices are included as headers in all our public repositories.
We'll automatically check license headers with the [SkyWalking Eyes](https://github.com/apache/skywalking-eyes) GitHub action.

### Benefits

Automatic license compliance.

### Possible drawbacks

No known drawbacks.

### Applicable issues

None.

### Checklist

- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by agarcia-oss** [comment]
I think this PR should also update the [CONTRIBUTING](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md) guidelines since it includes the restriction of license headers. WDTY?

File: bitnami/clickhouse/Chart.yaml [hunk 1]
```diff
@@ -28,4 +28,4 @@ maintainers:
 name: clickhouse
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/clickhouse
-version: 3.5.1
+version: 3.5.2
```
Comment (carrodher) [inline bitnami/clickhouse/Chart.yaml:31]:
```suggestion
version: 3.5.1
```

File: bitnami/jupyterhub/Chart.yaml [hunk 2]
```diff
@@ -28,4 +28,4 @@ maintainers:
 name: jupyterhub
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/jupyterhub
-version: 4.1.6
+version: 4.1.7
```
Comment (carrodher) [inline bitnami/jupyterhub/Chart.yaml:31]:
```suggestion
version: 4.1.6
```

**Review by carrodher — COMMENTED** [review]
Thanks, Raul! Given that this contribution is a little bit special let's try to simplify the process.

Usually, it is required to create a single PR per chart but in this case, we are modifying a couple of Helm charts in a single PR, this will cause issues when testing/releasing the Helm charts since the test & release pipelines only expect a single chart.
Given said that, we can avoid the version bump in the _Chart.yaml_ to avoid triggering a new release. This is possible because the nature of the change is not modifying the templates themselves, just adding a comment. If we don't bump the chart version, the changes will be present in the GitHub code but we won't publish a new Helm chart.

TL;DR
By reverting the version bump there will be a failure in one of the GH actions but the whole test & release process will be smooth, mainly because a release won't be triggered.

File: .github/workflows/license-headers.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Copyright VMware, Inc.
+# SPDX-License-Identifier: APACHE-2.0
+
+---
+name: license-headers
```
Comment (carrodher) [inline .github/workflows/license-headers.yml:5]: We usually add a prefix to order the actions (i.e `[CI/CD] ...`, `[Support] ...`, etc) and a descriptive title

What do you think about using something like this?
```suggestion
name: [License] Check license headers
```

File: .licenserc.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      spdx-id: Apache-2.0
+      copyright-owner: VMware, Inc.
+
+      content: |
+        {{- /*
+        Copyright VMware, Inc.
+        SPDX-License-Identifier: APACHE-2.0
+        */}}
+
+    paths:
```
Comment (carrodher) [inline .licenserc.yaml:18]: There are also a `_helpers.tpl` file present in most Helm charts that would be nice to check as well, see https://github.com/bitnami/charts/blob/main/bitnami/postgresql/templates/_helpers.tpl

File: .licenserc.yaml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+      - "bitnami/**/templates/**/*.yml"
+      - "bitnami/**/templates/**/*.yaml"
```
Comment (carrodher) [inline .licenserc.yaml:20]: Are we contemplating files directly in `/templates` with that approach or just files under another subdirectory `templates/foo`?
I.e `bitnami/postgresql/templates/read/svc.yaml` and `bitnami/postgresql/templates/svc.yaml`

**Comment by carrodher** [comment]
PS: Failure in the _"Get modified charts"_ GH action is expected, we can merge the PR with this error

File: .licenserc.yaml [hunk 6]
```diff
@@ -0,0 +1,37 @@
+# Copyright VMware, Inc.
+# SPDX-License-Identifier: APACHE-2.0
+
+---
+header:
```
Comment (andresbono) [inline .licenserc.yaml:5]: Should we cover [.vib](https://github.com/bitnami/charts/tree/main/.vib) and [template](https://github.com/bitnami/charts/tree/main/template) in this PR or in a follow-up?

Comment (rloporp) [reply .licenserc.yaml:5]: I agree we should double-check several other files, but we're limiting the scope to YAML files (`Chart.yaml`, `values.yaml` and Helm templates) in this first iteration.

File: .licenserc.yaml (hunk 5 above)
Comment (carrodher) [reply .licenserc.yaml:20]: 👍 Thanks

File: .licenserc.yaml (hunk 4 above)
Comment (carrodher) [reply .licenserc.yaml:18]: Perfect, totally fine

**Review by carrodher — APPROVED** [review]
LGTM
